src/components/ChatBeginingText.js

```diff
@@ -0,0 +1,60 @@
+import React, {memo} from 'react';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import Text from './Text';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import compose from '../libs/compose';
+
+const propTypes = {
+
+    isDefaultChatRoom: PropTypes.bool,
+    chatUsers: PropTypes.arrayOf(PropTypes.object),
+    ...withLocalizePropTypes,
+
+};
+
+const defaultProps = {
+    chatUsers: [],
+    isDefaultChatRoom: false,
+};
+
+const ChatBeginingText = ({isDefaultChatRoom, chatUsers, translate}) => (
+    <Text style={[styles.mt3, styles.w70, styles.textAlignCenter]}>
+        <Text>
+            {isDefaultChatRoom ? `${translate('reportActionsView.beginingOfChatHistroyPrivate')} ` : `${translate('reportActionsView.beginingOfChatHistroy')} `}
+        </Text>
+        {isDefaultChatRoom
+            && (
+                <Text>
+                    {`${chatUsers?.[0]?.displayName} ${translate('reportActionsView.beginingOfChatHistroyPrivateSectionPart')}`}
```

Comment:
You can use functions Translation, See examples on en.js. There are few keys there are functions.

Reply:
This style `?` is not encouraged in this codebase. Please see the styleGuide. You can use LodashGet.

Reply:
ok i got it.